### PR TITLE
Implement set/get methods in CalendarWrapper for color and calendar_id

### PR DESCRIPTION
**Pull request by Anton-Lazarev**

### CODE REVIEW

**Comment by Anton-Lazarev** [comment]
@vasily-v-ryabov 
New pull request don't resolve problem. How expected.
My fork in web tell me that:
image

My local repo tell me: 
image

My repo in SourceTree view like this:
image

I think that problem in source tree repo, but my knowledge don't enough for finding problem.
May you help me, please? Or best and fastest it create new account on Git? :) 
Do I able to "reFork" this project for rollback to start point without my commits or I should clear my commits?

**Comment by vasily-v-ryabov** [comment]
Did you make `git push`? Just in case.

**Comment by Anton-Lazarev** [comment]
Yes. Try it many times. Bash tell me that's all everything updated.
image

File: pywinauto/win32defines.py [hunk 1]
```diff
@@ -12010,7 +12010,7 @@
 MCMV_CENTURY = 3
 
 MCM_SETCALID = 0 # Variable c_int
-MCM_GETCALID = 1 # Variable c_int
+MCM_GETCALID = 0 # Variable. c_int
```
Comment (vasily-v-ryabov) [inline pywinauto/win32defines.py]: It seems `MCM_GETCALID` must equal to `MCM_FIRST + 25` ([proof link](http://www.rdos.net/svn/tags/V9.2.5/watcom/bld/sdk/spy/spyexmsg.h)). Almost the same for `MCM_SETCALID`.

**Comment by Anton-Lazarev** [comment]
@vasily-v-ryabov fix win32defines.py

**Pull request merged**